### PR TITLE
Vulkan: enable cull face again

### PR DESCRIPTION
**Pull request by raven02**
fix https://github.com/RPCS3/rpcs3/issues/2095

### CODE REVIEW

**Comment by raven02** [comment]
Need to do more testing in this PR as this break Project Diva F previously.

**Comment by Nekotekina** [comment]
How much % of required functionality is implemented?

**Comment by raven02** [comment]
Just tested Project Diva F . It looks okay .

**Comment by raven02** [comment]
@Nekotekina . i think should be at least 90% implemented the cull face stuff.

**Comment by raven02** [comment]
@kd-11 , looks okay to you?

File: rpcs3/Emu/RSX/VK/VKGSRender.cpp [hunk 1]
```diff
@@ -265,13 +265,13 @@ namespace vk
...
 		{
-		case CELL_GCM_FRONT: return VK_CULL_MODE_FRONT_BIT;
-		case CELL_GCM_BACK: return VK_CULL_MODE_BACK_BIT;
-		case CELL_GCM_FRONT_AND_BACK: return VK_CULL_MODE_FRONT_AND_BACK;
+		case rsx::cull_face::back: return VK_CULL_MODE_BACK_BIT;
+		case rsx::cull_face::front: return VK_CULL_MODE_FRONT_BIT;
+		case rsx::cull_face::front_and_back: return VK_CULL_MODE_FRONT_AND_BACK;
 		default: return VK_CULL_MODE_NONE;
 		}
 		fmt::throw_exception("Unknown cull face value: 0x%x" HERE, (u32)cfv);
```
Comment (paulsapps) [inline rpcs3/Emu/RSX/VK/VKGSRender.cpp]: this line is dead because the switch has a default case

**Comment by kd-11** [comment]
The original issue with face culling and project diva F was fixed. Culling can be re-enabled but there's the issue of rsx::window_origin that is unimplemented and untested using this backend.

File: rpcs3/Emu/RSX/VK/VKGSRender.cpp (hunk 1 above)
Comment (raven02) [inline rpcs3/Emu/RSX/VK/VKGSRender.cpp]: thanks .Will fix it.

**Comment by raven02** [comment]
@kd-11 . I'll check out the rsx::window_origin shortly .

**Pull request merged**